tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/Constants/RateLimitConstants.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Sdk.Tools.GitHubEventProcessor.Constants
+{
+    public class RateLimitConstants
+    {
+        // https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#rate-limits-for-requests-from-github-actions
+        // When using the GITHUB_TOKEN the number of Actions per hour for an Enterprise repository is 15000/hour
+        public const int ActionRateLimitEnterprise = 15000;
```

Comment:
Given that we call the rate limit API at the start of this workflow anyway, do we want to just set these values dynamically based on the result?